engine/src/main/java/com/datatorrent/stram/api/StreamingContainerUmbilicalProtocol.java

```diff
@@ -413,7 +413,7 @@ public String toString()
    * @param operators
    * @param msg
    */
-  void reportError(String containerId, int[] operators, String msg);
+  void reportError(String containerId, int[] operators, String string, String logFileName, long logFileOffset);
```

Comment:
I am returning default values as filename=null and offset=0, if right appender is not used.
Even if I skip to emit filename, the ContainerErrorEvent and OperatorErrorEvent class beans are updated to have getter and setter for filename and offset. Due to which those fields will be read (check FSEventRecorder "writeEvent" function.). It uses BeanUtils.describe function and hence calls all getter.
So I decided to add default values than skipping it.